### DISCUSSION TITLE
Monetary Contributions

### DISCUSSION THREAD

**Discussion by brickhousetech** [Jxck-S/plane-notify#108] [Ideas]
Hi Jack,

Posting this here because it isn't an issue, but rather asking for a way to contribute other than code.  I'm sure everyone who has joined in this appreciates the fact you've made this project public and allow others to use your code.  It also seems like you have plans to improve upon what you've done.  I realize the coding community is made up of people like yourself who do this for the joy of the challenge; however, I think you should consider posting a way to contribute financially to your efforts.  Not to commercialize what you've done, but allow those of us using your work to contribute and donate to you.  I know the APIs cost you money.  

Please consider this a request to post a Paypal or other method of making donations to your efforts.

**Comment by Cfomodz** [comment]
https://grndcntrl.net/donate/ :) Thank you for contributing to open source!
